flexmeasures/ui/templates/views/status.html

```diff
@@ -55,6 +55,88 @@ <h3>Data connectivity for sensors of {{ asset.name }}</h3>
                         {% endfor %}
                     </tbody>
                 </table>
+                <h3>Latest scheduling jobs of {{ asset.name }}</h3>
+                <table class="table table-striped table-responsive paginate nav-on-click">
+                    <thead>
+                        <tr>
+                            <th>Job id</th>
+                            <th>Asset id</th>
+                            <th>Asset type</th>
+                            <th class="text-right no-sort">Status</th>
+                            <th class="hidden">URL</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for job in scheduling_jobs: %}
+                        <tr title="View data">
+                            <td>
+                                {{ job.job_id }}
+                            </td>
+                            <td>
+                                {{ job.asset_id }}
+                            </td>
+                            <td>
+                                {{ job.asset_type }}
+                            </td>
+                            <td class="text-right">
+                                <span title="{{ job['err'] is none and job['status'] or job['status'] + ' : ' + job['err'] }}">
+                                    {% if job["status"] == "finished" %}
+                                    🟢
+                                    {% elif job["status"] == "failed" %}
+                                    🔴
+                                    {% else %}
+                                    🟡
+                                    {% endif %}
+                                </span>
+                            </td>
+                            <td class="hidden">
+                                https://seita.energy/tasks/0/view/job/{{ job.job_id }}
```

Comment:
This should not be hard-coded to point to "https://seita.energy/".

It should find the root URL (with `url_for("index")`, by the `request` object, or even crudely with "../../tasks/0" etc.

I didn't test those ideas, so I don't know what the go-to method would be.